docs/detections/rules-ui-create.asciidoc

```diff
@@ -259,6 +259,42 @@ IMPORTANT: When checking multiple fields, each unique combination of values from
 For example, if a rule has an interval of 5 minutes, no additional look-back time, and a history window size of 7 days, a term will be considered new only if the time it appears within the last 7 days is also within the last 5 minutes. Configure the rule interval and additional look-back time when you <<rule-schedule, set the rule's schedule>>.
 . Click *Continue* to <<rule-ui-basic-params, configure basic rule settings>>.
 
+[discrete]
+[[create-esql-rule]]
```

Comment:
would we have a separate page dedicated to ES|QL later?

Reply:
Sure, @vitaliidm! We can set one up for 8.11 or a future release. What do you have in mind for the separate page? Do you want to create a separate page that walks users through several types of ES|QL rule queries or maybe something different?

Reply:
Quick update from Slack convo with Vitalii today: We're going to revisit this convo next Wednesday and will update the [tooltip text/link](https://github.com/elastic/ux-writing/issues/41#issuecomment-1747957148) if needed.

Reply:
Resolving because moved stub to new page that'll provide ample space for example queries.